src/main/webapp/js/pages/index.js

```diff
@@ -24,8 +24,19 @@ document.querySelector('.course-list').addEventListener('click', () => {
   Calendar.clear();
   const selected = CollegePlanner.getSelected();
   const courseInfo = CollegePlanner.getCourseInfo();
+  // TODO{naaoli}: connect to algorithm servlet
+  // hard code return from algorithm servlet
+  const sections1 = {};
+  selected.forEach(
+      course => {sections1[course] = courseInfo[course].sections[0]});
+  const sections2 = {};
+  selected.forEach(
+      course => {sections2[course] = courseInfo[course].sections[1]});
+  const schedules = [sections1, sections2];
+
+  // TODO{ramyabuva}: add pagination for multiple schedules
```

Comment:
do these 2 todos have associated github issues?

Reply:
not yet, i'll create them rn